4337/.env.sample

```diff
@@ -19,6 +19,9 @@ SCRIPT_ADD_MODULES_LIB_ADDRESS=""
 SCRIPT_PROXY_FACTORY_ADDRESS=
 # Singleton address to use in the user op test script
 SCRIPT_SAFE_SINGLETON_ADDRESS=
+# ERC20 Token to use in the user op test script for testing contract calls. Token should have 18 decimals. Safe should have 1 token.
+# Optional. If set, the runOp script will execute a token transfer out of the Safe
+SCRIPT_ERC20_TOKEN_ADDRESS=
```

Comment:
nitpick: Not related to this PR, but I noticed some slight inconsistencies in this file when declaring variables with empty values (some are `VAR=""` and some are `VAR=`); might be nice to clean up for consistency.